ui-ngx/src/app/modules/home/components/widget/lib/rpc/knob.component.ts

```diff
@@ -392,7 +392,9 @@ export class KnobComponent extends PageComponent implements OnInit, OnDestroy {
     this.ctx.detectChanges();
   }
 
-  private updateColor(color: string) {
+  private updateColor(color: string) {  
+    this.minmaxLabel.css({color: this.ctx.widgetConfig.color});
+    this.knobTitle.css({color: this.ctx.widgetConfig.color});
```

Comment:
This code does not support backward compatibility, the default color is text in widget settings rgba(0,0,0,0.87), but the current color has #757575